run_tests.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Is it worth adding a note about this to README?

Reply:
added!